Underanalyzer/VMData.cs

```diff
@@ -585,12 +606,12 @@ public enum VariableType : byte
         Instance = 0xE0,
 
         /// <summary>
-        /// Used in tandem with multi-dimensional array push operations ("pushaf" extended opcode).
+        /// Used in tandem with multi-dimensional array push operations (<c>pushaf</c> extended opcode).
         /// </summary>
         MultiPush = 0x10,
 
         /// <summary>
-        /// Used in tandem with multi-dimensional array push and pop operations ("pushaf"/"popaf" extended opcodes).
+        /// Used in tandem with multi-dimensional array push and pop operations (<c>pushaf</c>/<c>popaf</c> extended opcodes).
```

Comment:
Maybe for consistency with the new changes, these should become `ExtendedOpcode.PushArrayFinal` and `ExtendedOpcode.PopArrayFinal` - same with line 609 as well (above).